doc/rst/_static/style.css

```
@@ -1,38 +1,48 @@
 /* Set max width of the content */
 .wy-nav-content {
-    max-width: 1000px;
+	max-width: 1000px;
 }
 
 /* See https://github.com/GenericMappingTools/gmt/pull/1605 */
 .rst-content dl dt {
-    font-weight: normal;
+	font-weight: normal;
+}
+
+body {
+	font-family: "Atkinson Hyperlegible", sans-serif;
+	font-size: 1.05em;
+}
+
+h1, h2, h3, h4, h5, h6 {
+	font-family: "Atkinson Hyperlegible";
```

Shouldn't there be a fallback font here?

```suggestion
	font-family: "Atkinson Hyperlegible", sans-serif;
```